docs/Vue.js/TransitionAndAnimation.md

```diff
@@ -0,0 +1,117 @@
+---
+id: Vue.js Transition and animation
```

Comment:
change `id` in good format because we are using `id` for unique and for routing like `url` so, write in right way for SEO and best practice